Helper/PaymentResponseHandler.php

```diff
@@ -105,6 +105,7 @@ public function formatPaymentResponse($resultCode, $action = null, $additionalDa
                 return [
                     "isFinal" => true,
                     "resultCode" => $resultCode,
+                    "donationToken" => $donationToken
```

Comment:
`donationToken` parameter should be include in the response for only `AUTHORISED` and `RECEIVED` event codes.